doc/script_commands.txt

```diff
@@ -4201,6 +4201,7 @@ SavePoint:    All party members are warped to the save point of the currently
               attached player (will fail if there's no player attached).
 Leader:       All party members are warped to the leader's position. The leader must
               be online and in the current map-server for this to work.
+RandomAll:    All party members are warped to the same random position in their current map
```

Comment:
```suggestion
RandomAll:    All party members are warped to a random position in their current map
```

Reply:
I phrased it that way to avoid confusion between Random and RandomAll, because in this case all players are warped to the same random cell.